gef-ui/server/utils/display-items.test.js

```diff
@@ -0,0 +1,96 @@
+import { facilityItems } from './display-items';
+import { MOCK_FACILITY } from './mocks/mock_facilities';
+
+describe('facilityItems', () => {
+  it('should show the facility end date row when the facility end date exists', () => {
+    // Arrange
+    const facility = { ...MOCK_FACILITY, facilityEndDateExists: true };
+
+    // Act
+    const result = facilityItems('testUrl', facility);
+
+    // Assert
+    expect(result.find((item) => item.id === 'facilityEndDate').isHidden).toEqual(false);
+  });
+
+  it('should hide the bank review date row when the facility end date exists', () => {
+    // Arrange
+    const facility = { ...MOCK_FACILITY, facilityEndDateExists: true };
+
+    // Act
+    const result = facilityItems('testUrl', facility);
+
+    // Assert
+    expect(result.find((item) => item.id === 'bankReviewDate').isHidden).toEqual(true);
+  });
+
+  it('should hide the facility end date row when the facility end date does not exist', () => {
+    // Arrange
+    const facility = { ...MOCK_FACILITY, facilityEndDateExists: false };
+
+    // Act
+    const result = facilityItems('testUrl', facility);
+
+    // Assert
+    expect(result.find((item) => item.id === 'facilityEndDate').isHidden).toEqual(true);
+  });
+
+  it('should show the bank review date row when the facility end date does not exist', () => {
+    // Arrange
+    const facility = { ...MOCK_FACILITY, facilityEndDateExists: false };
+
+    // Act
+    const result = facilityItems('testUrl', facility);
+
+    // Assert
+    expect(result.find((item) => item.id === 'bankReviewDate').isHidden).toEqual(false);
+  });
+
+  it('should hide the facility end date row when the facility end date is null', () => {
```

Comment:
```suggestion
  it('should hide the facility end date row when facilityEndDateExists is null', () => {
```

Reply:
Is this going to be null, or is it going to be undefined?